src/features/hotspots/settings/ReassertAddressSearch.tsx

```diff
@@ -99,7 +101,7 @@ const ReassertAddressSearch = ({ onSelectPlace }: Props) => {
   )
 
   return (
-    <Box height={hp(75)} padding="none" paddingTop="lx">
+    <Box height={600} padding="none" paddingTop="lx">
```

Comment:
Is this too tall for some phones?

Reply:
I think it should be ok, when we use the hp I notice things sometimes differ but fixed pixel is supposed to be density independent so it should always be the same height on any device. Let me try it on a small phone though

Reply:
It is the same height but its taller than the device on the SE lol. Ill make it smaller for small devices